component/addons/disable-alerts.libsonnet

```
@@ -0,0 +1,55 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local inv = kap.inventory();
+local params = inv.parameters.prometheus;
+
+local ruleFilter(group) =
+  local internalRuleFilter =
+    if group.name == 'kubernetes-apps' then
+      [ 'KubeHpaMaxedOut', 'KubeHpaReplicasMismatch' ]
+    else if group.name == 'kubernetes-resources' then
+      [ 'CPUThrottlingHigh' ]
+    else
+      [];
+
+  // merge user-provided and hard-coded rule names to filter out
+  local ignoreSet = std.set(internalRuleFilter + params.addon_configs.disable_alerts.ignoreNames);
```

I'd use [`com.renderArray()`](https://syn.tools/commodore/reference/commodore-libjsonnet.html#_renderarrayarr) here, so that users can remove ignored alerts from the list:

```suggestion
  local ignoreSet = std.set(com.renderArray(internalRuleFilter + params.addon_configs.disable_alerts.ignoreNames));
```

This needs an `import com = 'lib/commodore.libjsonnet';` in the file.